### ISSUE TITLE
refactor config paramerters

### ISSUE THREAD

**Issue by sibalzer** [sibalzer/impfbot#45]
refactor config paramerter names without breaking compability with older versions

**Comment by sibalzer** [comment]
First view of the new config key names:

```
SKELETON_CONFIG = {
    "COMMON": {
        "zip": str,
        "birthdate": str,
    },
    "EMAIL": {
        "enable": bool,
        "sender": str,
        "password": str,
        "server": str,
        "port": int,
        "receivers": List[str],
    },
    "TELEGRAM": {
        "enable": bool,
        "token": str,
        "chat_ids": [List[str],
    },
[ADVANCED]:
    "cooldown_between_requests",
    "cooldown_between_failed_requests",
    "cooldown_after_ip_ban",
    "cooldown_after_success",
    "jitter",
    "sleep_at_night",
}
```

**Comment by paulypeter** [comment]
> First view of the new config key names:

What about the `[WEBBROWSER]` settings?

**Comment by sibalzer** [comment]
Was missing and added in (d21bb89)

**Comment by sibalzer** [comment]
closed with #44
